client/src/components/DataEntry/BallotAudit.tsx

```diff
@@ -32,14 +32,14 @@ const BallotAudit: React.FC<IProps> = ({
           marked on the paper ballot.
         </p>
         <p>
-          If the voter did not vote in the contest, select &quot;Blank vote/Not
-          on Ballot.&quot;
+          If the voter did not vote in the contest, select &quot;Blank
+          vote.&quot;
```

Comment:
Should we add instructions for the "Not on ballot" case? Maybe:

If this contest is not shown on this ballot, select "Not on ballot"

Reply:
I'll do that, and if @mcchilders has a better suggestion we can do that in another PR.